web/src/buddy_list.ts

```diff
@@ -457,22 +457,15 @@ export class BuddyList extends BuddyListConf {
             return;
         }
 
-        let header_text;
-        if (current_sub) {
-            if (all_participant_ids.size) {
-                header_text = $t({defaultMessage: "OTHERS IN THIS CHANNEL"});
-            } else {
-                header_text = $t({defaultMessage: "IN THIS CHANNEL"});
-            }
-        } else {
-            header_text = $t({defaultMessage: "IN THIS CONVERSATION"});
-        }
+        const header_text = current_sub
+            ? $t({defaultMessage: "THIS CHANNEL"})
+            : $t({defaultMessage: "THIS CONVERSATION"});
```

Comment:
Maybe this should be defined before the one place it's used, one section down?

Reply:
This can be a follow-up, since it's a nonfunctional change, and I'm not sure this alone is the right refactor; we might instead want to have the "THIS CONVERSATION" DMs heading be done a bit differently.